gotpm/cmd/seal_test.go

```diff
@@ -104,17 +121,26 @@ func TestUnsealFail(t *testing.T) {
 			sealedFile := makeTempFile(t, nil)
 			defer os.Remove(sealedFile)
 
-			RootCmd.SetArgs([]string{"seal", "--quiet", "--input", secretFile, "--output", sealedFile, "--pcrs", "23"})
+			sealArgs := []string{"seal", "--quiet", "--input", secretFile, "--output", sealedFile}
+			if test.sealPCRs != "" {
+				sealArgs = append(sealArgs, "--pcrs", test.sealPCRs)
+			}
+			RootCmd.SetArgs(sealArgs)
 			if err := RootCmd.Execute(); err != nil {
 				t.Error(err)
 			}
+			initPCRs()
 
-			if err := test.tpmFunc(rwc); err != nil {
+			if err := test.tpmFunc(rwc, test.tpmFuncParameter); err != nil {
 				t.Fatal(err)
 			}
+			unsealArgs := []string{"unseal", "--quiet", "--input", sealedFile, "--output", secretFile}
+			if test.certifyPCRs != "" {
+				unsealArgs = append(unsealArgs, "--certify-pcrs", test.certifyPCRs)
+			}
 
-			RootCmd.SetArgs([]string{"unseal", "--quiet", "--input", sealedFile, "--output", secretFile})
-			if RootCmd.Execute() == nil {
+			RootCmd.SetArgs(unsealArgs)
+			if err := RootCmd.Execute(); err == nil {
```

Comment:
Why do you need to add err, if you aren't going ot use it?

Reply:
Removed. Used when debugging, forget to clean.